### DISCUSSION TITLE
VIEW with ID + FTS fails to match corresponding match to its table.

### DISCUSSION THREAD

**Discussion by pcardosolei** [supabase/supabase#9331] [Questions]
Hello.

I created a VIEW that just contains id of a table and its corresponding FTS.

When I use the supabase-js to query that view + table it says that I have no relationships.

small snippet.
```
supabase
    .from<TableWFTS>(FTS_VIEW_NAME)
    .select(`id, table!inner(*)`)
```

fails to deliver. any guidance here?

**Comment by GaryAustin1** [comment]
Possibly here: https://postgrest.org/en/stable/api.html#embedding-views
You might provide more info on your view setup and table structure if that is not of help.

**Reply by pcardosolei** [reply]
I'm not sure that is the problem. the view is pretty much a combination results are just id and the fts results and it doesnt manage to get the key when comparing to the jobs. is there is a way to specify the ON match between view and jobs?

```
CREATE OR REPLACE VIEW jobs_fts AS
SELECT jobs.id, to_tsvector(
'english', concat_ws(' ', jobs.title, orgs.name, jobs.description, COALESCE(immutable_array_to_string (jobs.category, ' ')))) AS fts 
FROM jobs JOIN orgs ON jobs.org_id = orgs.id
```
